invenio_vocabularies/administration/views/vocabularies.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020-2021 CERN.
+# Copyright (C) 2024 Uni Münster.
+#
+# Invenio-Vocabularies is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Vocabularies admin interface."""
+from flask import current_app
+from invenio_administration.views.base import AdminResourceListView
+
+
+class VocabulariesListView(AdminResourceListView):
+    """Configuration for vocabularies list view."""
+
+    api_endpoint = "/vocabularies/"
+    name = "vocabulary-types"
+    menu_label = "Vocabulary Types"
+    resource_config = "vocabulary_admin_resource"
+    search_request_headers = {"Accept": "application/json"}
+    title = "Vocabulary Types"
+    category = "Site management"
+
+    pid_path = "id"
+    icon = "exchange"
+    template = "invenio_administration/search.html"
+
+    display_search = True
+    display_delete = False
+    display_edit = False
+    display_create = False
+
+    item_field_list = {
+        "id": {"text": "Name", "order": 1},
+        "count": {"text": "Number of entries", "order": 2},
+    }
+
+    search_config_name = "VOCABULARIES_TYPES_SEARCH"
+    search_facets_config_name = "VOCABULARIES_TYPES_FACETS"
+    search_sort_config_name = "VOCABULARIES_TYPES_SORT_OPTIONS"
+
+
+class VocabularyDetailsListView(AdminResourceListView):
```

Comment:
I noticed DetailView inherits from ListView here, is there a reason? I think it might cause some issues

Reply:
Not yet. Could you explain how that might help with our issues?

Reply:
it would ensure less ambiguity on where the methods are actually coming from, this class now only imitates the interface of AdminResourceDetails view in order to use some of its frontend functionalities, while it does not actually inherit from the class itself. It might create divergence on the implementation of these "virtually" inherited methods the code it will be harder to maintain.
So to answer your question, it does not solve immediate problems but prevents ambiguity in the future